core/util/model/Logger.py

```diff
@@ -4,41 +4,38 @@
 
 class Logger:
 
-	def __init__(self, depth: int = 4, *args, **kwargs):
+	def __init__(self, depth: int = 4):
 		self._logger = logging.getLogger('ProjectAlice')
 		self._depth = depth
 
 
-	def logInfo(self, msg: str):
+	def info(self, msg: str):
 		self.doLog(function='info', msg=msg, printStack=False)
 
 
-	def logError(self, msg: str):
+	def error(self, msg: str):
 		self.doLog(function='error', msg=msg)
 
 
-	def logDebug(self, msg: str):
+	def debug(self, msg: str):
 		self.doLog(function='debug', msg=msg, printStack=False)
 
 
-	def logFatal(self, msg: str):
+	def fatal(self, msg: str):
 		self.doLog(function='fatal', msg=msg)
 
 
-	def logWarning(self, msg: str, printStack: bool = False):
+	def warning(self, msg: str, printStack: bool = False):
 		self.doLog(function='warning', msg=msg, printStack=printStack)
 
 
-	def logCritical(self, msg: str):
+	def critical(self, msg: str):
 		self.doLog(function='critical', msg=msg)
 
 
-	def doLog(self, function: callable, msg: str, printStack=True, depth: int = None):
-		if depth is None:
-			depth = self._depth
-
+	def doLog(self, function: callable, msg: str, printStack=True):
 		func = getattr(self._logger, function)
```

Comment:
since depth is never provided in the logging method anyways it can be removed aswell